include/devicetree/clocks.h

```diff
@@ -12,6 +12,8 @@
 #ifndef ZEPHYR_INCLUDE_DEVICETREE_CLOCKS_H_
 #define ZEPHYR_INCLUDE_DEVICETREE_CLOCKS_H_
 
+#include <sys/util_macro.h>
+
```

Comment:
Sorry that I missed this last time, but can you explain why you are adding this include here? It doesn't seem to be needed.

Reply:
Good point, IIRC it was required initially while I was not based in devicetree.h macros.
Now this could be removed. Done